src/SdkConfig.ts

```diff
@@ -53,6 +56,7 @@ export const DEFAULTS: ConfigOptions = {
         logo: require("../res/img/element-desktop-logo.svg").default,
         url: "https://element.io/get-started",
     },
+    spaces_learn_more_url: "https://element.io/blog/spaces-blast-out-of-beta/",
```

Comment:
> I'm trepidatious about linking to the blog post. We can't guarantee the link will be active forever. I'd rather we add some content to [element.io/help](https://element.io/help) and link there instead. Does that work for you? If so I can make that happen and we can include in another PR.

I don't think making it configurable solves this, given we have no control over the configs of other deployments and the link could break there.

Reply:
The link is mostly for branded clients, not the longevity of the link tbh.